ios/FluentUI/Button/ButtonTokenSet.swift

```diff
@@ -344,9 +344,9 @@ extension ButtonTokenSet {
             case .large:
                 return 52
             case .medium:
-                return 40
+                return Compatibility.isDeviceIdiomVision() ? 44 : 40
```

Comment:
How is this different from the iOS HIG, which also recommends 44pt high buttons on iOS?

Reply:
Actually, [per the HIG](https://developer.apple.com/design/human-interface-guidelines/buttons#Best-practices) even that's not quite right:
> As a general rule, a button needs a hit region of at least 44x44 pt — in visionOS, 60x60 pt — to ensure that people can select it easily, whether they use a fingertip, a pointer, their eyes, or a remote.

Reply:
that's hit area right? this is for the actual size of the button?

Reply:
Technically true, but our existing buttons all use the same hit and visual bounding boxes. And I'm not sure I'd want to change that except in specific one-off cases.